reports/final_report.qmd

```diff
@@ -17,7 +17,7 @@ format:
     number-sections: true
     colorlinks: true
 params: 
-  output_dir: "reports"
+  output_file: "reports"
```

Comment:
I think this can be removed completely since we found the --data-dir flag, shouldn't need any parameters, I'll fix this.